server/db/sequelize/models/users.js

```diff
@@ -0,0 +1,95 @@
+import Promise from 'bluebird';
+import bcryptNode from 'bcrypt-nodejs';
+const bcrypt = Promise.promisifyAll(bcryptNode);
```

Comment:
@mjewell Do you see value in adding `bluebird` as an added dependency as opposed to using `es6-promise`?

On one hand, it promisifies every function, but just being careful about adding another promise library (although bluebird appears to be faster hehe)